app/models/concerns/hashable.rb

```diff
@@ -1,11 +1,7 @@
 module Hashable
-  extend ActiveSupport::Concern
+  def hash_uuid(original_uuid)
+    return nil if original_uuid.blank?
 
-  included do
```

Comment:
I did this by hand -- there is no reason to use AS::Concern where a plain ole' module will do.